src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -103,32 +103,24 @@ const CourseEnrollments = ({ children }) => {
   }
 
   return (
-    <>
+    <CourseEnrollmentsContext.Provider value={courseEnrollmentsContextValue}>
       {enterpriseFeatures.enterpriseGroupsV1 && (
         <NewGroupAssignmentAlert
           showAlert={showNewGroupAssociationAlert}
           onClose={dismissGroupAssociationAlert}
           enterpriseCustomer={enterpriseCustomer}
         />
       )}
-      {shouldShowMarkSavedForLaterCourseSuccess && (
-        <CourseEnrollmentsAlert variant="success" onClose={() => setShouldShowMarkSavedForLaterCourseSuccess(false)}>
-          <FormattedMessage
-            id="enterprise.dashboard.course.enrollment.saved.for.later.alert.text"
-            defaultMessage="Your course was saved for later."
-            description="Message when a course is saved for later."
-          />
-        </CourseEnrollmentsAlert>
-      )}
-      {shouldShowMoveToInProgressCourseSuccess && (
-        <CourseEnrollmentsAlert variant="success" onClose={() => setShouldShowMoveToInProgressCourseSuccess(false)}>
-          <FormattedMessage
-            id="enterprise.dashboard.course.enrollment.moved.to.progress.alert.text"
-            defaultMessage="Your course was moved to In Progress."
-            description="Message when a course is moved to In Progress."
-          />
-        </CourseEnrollmentsAlert>
-      )}
+      {courseEnrollmentsContextValue.courseEnrollmentStatusChanges?.length > 0
+        && courseEnrollmentsContextValue.courseEnrollmentStatusChanges.map(({ uuid, message, dismiss }) => (
```

Comment:
nit: this could be simplified

```suggestion
      {courseEnrollmentsContextValue.courseEnrollmentStatusChanges?.map(({ uuid, message, dismiss }) => (
```

We only really need the optional chaining; the `.map` is simply a no-op if the array exists but is empty.